tests/requirements.txt

```diff
@@ -0,0 +1,115 @@
+Django==2.2.6
```

Comment:
Can we reuse one or more of the files in ``/requirements/`` like ``common.txt`` and ``dev.txt``?  Otherwise the list of packages may become out-of-sync with the rest of our deployments.

Reply:
On the previous PR I raised the same concern and I agreed to go with it knowing that it could happen. In the future, I plan to make sure we move to poetry and prevent versions from falling apart.

Reply:
@camd  The specific versions in the production requirements files do not seem to work on my development machine.

This may be the problem

```
ERROR: THESE PACKAGES DO NOT MATCH THE HASHES FROM THE REQUIREMENTS FILE. If you have updated the package versions, please update the hashes. Otherwise, examine the package contents carefully; someone may have tampered with them.
    multidict==4.5.2 from https://files.pythonhosted.org/packages/38/c7/07d8d88c3c16fe65d8596da429d104229dd7433b07432694a3ab45a72eaa/multidict-4.5.2-cp37-cp37m-win_amd64.whl#sha256=4a6ae52bd3ee41ee0f3acf4c60ceb3f44e0e3bc52ab7da1c2b2aa6703363a3d1 (from -r requirements/common.txt (line 263)):
        Expected sha256 c18498c50c59263841862ea0501da9f2b3659c00db54abfbf823a80787fde8ce
             Got        4a6ae52bd3ee41ee0f3acf4c60ceb3f44e0e3bc52ab7da1c2b2aa6703363a3d1
```

Reply:
You need to add the hashes for Windows.

When I tried to set things up outside of the Docker container I had to look up some Mac specific hashes.

Reply:
@armenzg it may not be just the hashes:  the specific versions may be a problem on windows.  I do not know until I try the many combinations; if the version requirements are different on windows, then what do we do?

Reply:
I don't know. Do you have a specific example?